lib/gateway.js

```diff
@@ -146,6 +144,25 @@ function defineResolvers (schema, typeToServiceMap, serviceMap, typeFieldsToServ
   }
 }
 
+function makeEntityTransformData (field) {
+  const isNonNull = field.astNode.type.kind === Kind.NON_NULL_TYPE
+  const leafKind = isNonNull ? field.astNode.type.type.kind : field.astNode.type.kind
```

Comment:
There may be a better/more generic way to find theses values but for now it solves the issue nullable/not nullable lists